default/scripting/specials/ASTEROID_COATING_OWNED.focs.txt

```diff
@@ -1,30 +0,0 @@
-Special
-    name = "ASTEROID_COATING_OWNED_SPECIAL"
-    description = "ASTEROID_COATING_OWNED_SPECIAL_DESC"
-    stealth = 5
```

Comment:
this is the stealth of the special, not the object with the special attached

Reply:
thanks got it. the two files workaround makes sense now